cmd/tegola/cmd/cache/worker.go

```diff
@@ -74,11 +74,6 @@ func seedWorker(tileBuffer *float64, overwrite bool) func(ctx context.Context, m
 			}
 		}
 
-		//	set tile buffer if it was configured by the user
-		if tileBuffer != nil {
```

Comment:
This edit takes away functionality where the buffer can be set at the root of the config. We may deprecate this functionality in the future but we should continue supporting it for now to avoid a regression. `seedWorker` accepts a `tileBuffer`. You could add the following as a replacement:

```go
// check if a global tileBuffer has been set, and if so only set on the map if the default buffer is configured
if tileBuffer != nil && m.TileBuffer != tegola.DefaultTileBuffer {
  m.TileBuffer = tileBuffer
} 
```

Reply:
Thanks for pointing this out, I was mindlessly refactoring... It might be a bigger issue because:
`tileBuffer` is being set by the value in a configuration file (in the caller):
https://github.com/go-spatial/tegola/blob/be45ebde2dae7413f0970a631faffd8e6986c4ca/cmd/tegola/cmd/cache/seed_purge.go#L131-L135
and the map handles it's own tile buffer, as of #560 so, I'm thinking the `tileBuffer` parameter for this function is now unnecessary.

Reply:
The tile buffer can be set at the [config root](https://github.com/go-spatial/tegola/blob/master/config/config.go#L25) as well as the map. The patch I suggested maintains support for the config root approach, which we may eventually deprecate but not right now.

Reply:
I stand corrected, this is handled during config load: https://github.com/go-spatial/tegola/blob/master/config/config.go#L147

Reply:
After some extra-githubular discussions, it was decided to clean up `seedWorker` as part of this pull request as well